.claude/skills/math-unified/SKILL.md

```
@@ -154,28 +154,28 @@ I decide based on your request:
 ### Solve Equation
 ```
 User: Solve x² - 5x + 6 = 0
-Claude: uv run python "$CLAUDE_PROJECT_DIR/.claude/scripts/cc_math/sympy_compute.py" solve "x**2 - 5*x + 6" --var x
+Claude: uv run python "$CLAUDE_OPC_DIR/scripts/cc_math/sympy_compute.py" solve "x**2 - 5*x + 6" --var x
 Result: x = 2 or x = 3
 ```
 
 ### Compute Eigenvalues
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)